src/StockportWebapp/Views/stockportgov/Shared/GenericFeaturedItemList.cshtml

```diff
@@ -1,27 +1,33 @@
 ﻿
 @model StockportWebapp.Models.GenericFeaturedItemList
-@{
-    var take = string.IsNullOrEmpty(Model.ButtonText) ? Model.Items.Count : 8;
-    var noOfInitialTopics = 8;
-}
+
 <div class="primary-topics">
-    @for (var count = 0; count < noOfInitialTopics; count += 4)
+    @if (Model.HideButton == true)
```

Comment:
this can just be @(Model.HideButton)?

Reply:
also do not having the foreach affect the styling, as now each row of 4 items are not wrapped in a container div? if not then dont worry.

Reply:
Updated the if to remove '== true' and you're right the styling was affect, which is fixed